configs/default/FOSS-DRONIUSF7.config

```diff
@@ -0,0 +1,109 @@
+# Betaflight / STM32F7X2 (S7X2) 4.1.5 Mar 16 2020 / 05:21:46 (d4e74e39c) MSP API: 1.42
+# manufacturer_id: FOSS   board_name: DRONIUSF7   custom defaults: NO
```

Comment:
Can you please remove this line?

Reply:
Which one?  the first or second @mikeller

Reply:
The one that I commented on: `# manufacturer_id: [...]`.